LunaMary/src/Util.java

```diff
@@ -0,0 +1,22 @@
+public class Util {
+
+
+    public Util() {
+
+    }
+
+    public Boolean validateSexo(String data) {
+        boolean textValid = false;
+
+        if (!data.isEmpty() && data.replace(" ", "").length() == 1 && textValid == false) {
```

Comment:
Instead of using replace you could use trim() function which will remove spaces at the beginning and at the end of a String.